proto/p4info/Makefile.am

```diff
@@ -9,11 +9,17 @@ bin_PROGRAMS = pi_convert_p4info
 
 # convenience library to convert back-and-forth between p4info and the P4Info
 # protobuf message
-noinst_LTLIBRARIES = libpiconvertproto.la
+lib_LTLIBRARIES = libpiconvertproto.la
 
 libpiconvertproto_la_SOURCES = \
 p4info_to_and_from_proto.cpp \
-p4info_to_and_from_proto.h
+p4info_to_and_from_proto.h 
```

Comment:
this line can be removed

Reply:
ok, added a ws at the end, I'll fix it.

Reply:
the line is actually not needed because the file is now listed in `_HEADERS` so it is an implicit source

Reply:
ok